bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/sensor.xml

```diff
@@ -123,4 +102,121 @@
+	<channel-type id="sensorWakeup" advanced="true">
+		<item-type>String</item-type>
+		<label>Wakeup reason</label>
+		<description>Reasons which woke up the device: battery, button, periodic, poweron, sensor, alarm</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+
+	<channel-type id="sensorTemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Temperature from the sensor</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>CurrentTemperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="sensorExtTemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Temperature from the external sensor</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>CurrentTemperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="sensorExtHum">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Relative humidity in % (0..100%) from external sensor</description>
+		<category>Humidity</category>
+		<tags>
+			<tag>CurrentHumidity</tag>
+		</tags>
+		<state readOnly="true" pattern="%f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="sensorHumidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Relative humidity in % (0..100%)</description>
+		<tags>
+			<tag>CurrentHumidity</tag>
+		</tags>
+		<state readOnly="true" min="0" max="100" pattern="%f %unit%" />
+	</channel-type>
+	<channel-type id="sensorFlood">
+		<item-type>Switch</item-type>
+		<label>Flood Alarm</label>
+		<description>ON: Indicates flood condition / water detected</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="sensorSmoke">
+		<item-type>Switch</item-type>
+		<label>Smoke Alarm</label>
+		<description>ON: Indicates smoke detection</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="sensorLux">
+		<item-type>Number:Illuminance</item-type>
+		<label>Lux</label>
+		<description>Brightness from the sensor (Lux)</description>
+		<state readOnly="true" pattern="%f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="sensorIllumination">
+		<item-type>String</item-type>
+		<label>Illumination</label>
+		<description>Current illumination: dark/twilight/bright</description>
+		<state readOnly="true">
+			<options>
+				<option value="dark">Dark</option>
+				<option value="twilight">Twilight</option>
+				<option value="bright">Bright</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="sensorVibration">
+		<item-type>Switch</item-type>
+		<label>Vibration</label>
+		<description>ON: Vibration detected</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="sensorError">
+		<item-type>Text</item-type>
```

Comment:
I don't think this is a valid item type.

Reply:
fixed